it/api/endpoints/bfLoss/list/v3/ListBFLossesControllerISpec.scala

```diff
@@ -127,87 +58,188 @@ class ListBFLossesControllerISpec extends V3IntegrationBaseSpec {
 
     def request(): WSRequest = {
       setupStubs()
-      buildRequest(uri)
-        .addQueryStringParameters(queryParams: _*)
+      buildRequest(mtdUri)
+        .addQueryStringParameters(mtdQueryParams: _*)
         .withHttpHeaders(
           (ACCEPT, "application/vnd.hmrc.3.0+json"),
           (AUTHORIZATION, "Bearer 123") // some bearer token
         )
     }
+
+    val downstreamResponseJson: JsValue =
+      Json.parse(s"""
+                    |[
+                    |  {
+                    |    "incomeSourceId": "XAIS12345678911",
+                    |    "lossType": "INCOME",
+                    |    "broughtForwardLossAmount": 345.67,
+                    |    "taxYear": "2019",
+                    |    "lossId": "AAZZ1234567890A",
+                    |    "submissionDate": "2020-07-13T12:13:48.763Z"
+                    |  },
+                    |  {
+                    |    "incomeSourceId": "XAIS12345678912",
+                    |    "incomeSourceType": "04",
+                    |    "broughtForwardLossAmount": 385.67,
+                    |    "taxYear": "2020",
+                    |    "lossId": "AAZZ1234567890B",
+                    |    "submissionDate": "2020-08-13T12:13:48.763Z"
+                    |  }
+                    |]
+     """.stripMargin)
+
+    val responseJson: JsValue = Json.parse(s"""{
+                                              |  "losses": [
```

Comment:
can this all be moved further to the left? The `{` on line 91 could be on the next line, then everything moved left by 20 or 30 characters